test/testenv/s3utils.go

```diff
@@ -0,0 +1,75 @@
+package testenv
+
+import (
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+// Set S3 Variables
+var (
+	s3Region                  = os.Getenv("S3_REGION")
+	testS3Bucket              = os.Getenv("TEST_BUCKET")
+	testIndexesS3Bucket       = os.Getenv("TEST_INDEXES_S3_BUCKET")
+	enterpriseLicenseLocation = os.Getenv("ENTERPRISE_LICENSE_LOCATION")
+)
+
+// CheckPrefixExistsOnS3 lists object in a bucket
+func CheckPrefixExistsOnS3(prefix string) bool {
+	dataBucket := testIndexesS3Bucket
+	sess, err := session.NewSession(&aws.Config{Region: aws.String(s3Region)})
+	if err != nil {
+		logf.Log.Error(err, "Failed to create s3 session")
+	}
+	svc := s3.New(session.Must(sess, err))
```

Comment:
Nice! No issue here just wanted to tag @kumarajeet @vebken-splunk since this is what we could use for App framework.